### ISSUE TITLE
Native Github Issue Integration

### ISSUE THREAD

**Issue by mjy** [SpeciesFileGroup/taxonworks#172]
```
   when I am in the workbench 
      I can see a "issues" slideout
        when I fill it out 
            then a (labelled) issue is automatically logged to github
```

Issues: 
* authentication (need a TW GH bot?)
* See implementation on GBIF

**Comment by mjy** [comment]
Other considerations:

* Needs to be be a configurable, remember there are multiple instances of TW possible, not just one.  Not all instances may want to have reporting/issue tracking come here.
* May want to be able to label issues- e.g. one for software bugs/improvements/suggestsions, one for the scientific project(s).  This is likely overly complicated to start though.
* May be related to using GH to authenticate users in general.

**Comment by debpaul** [comment]
The instance of ARCTOS at Harvard implemented Bugzilla (instead of GitHub). It has a flavor very much like the way GBIF implemented it -- making it both easy to use (not intimidating), but still useful for the programmers. 

- What ever is chosen, it should be easy to use, and preferably from inside TW, not leaving to visit another site no matter if it's an feature request, or bug. 
- (Not sure if this is useful idea or not, but it's very easy to set up a google form that feeds GitHub. The Google Form could be linked inside the TW instance). 
  - It is possible to _customize the resulting labels_ on submission of a ticket using such a method. We did this with the Darwin Core Hour form.
- Not sure if Bugzilla is even an option, but I found it much easier (intuitive) to teach people to use, to produce useful tickets, than GitHub. It looks like GBIF might have gleaned their categories from this software.
